aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -61,9 +63,31 @@ HTTP/2 is enabled by default. Connections fall back to HTTP/1.1 if an HTTP/2 con
 
 ### Enable the IISIntegration components
 
-::: moniker range=">= aspnetcore-2.0"
+::: moniker range=">= aspnetcore-2.2"
+
+**In-process hosting model**
+
+A typical *Program.cs* calls <xref:Microsoft.AspNetCore.WebHost.CreateDefaultBuilder*> to begin setting up a host. `CreateDefaultBuilder` calls the <xref:Microsoft.AspNetCore.Hosting.WebHostBuilderIISExtensions.UseIISIntegration*> method to boot the [CoreCLR](/dotnet/standard/glossary#coreclr) and host the app inside of the IIS worker process (`w3wp.exe`). Performance tests indicate that hosting a .NET Core app in-process delivers higher request throughput compared to hosting the app out-of-process and proxying requests to [Kestrel](xref:fundamentals/servers/kestrel).
```

Comment:
UseIIS?

Reply:
For in-proc it's UseIIS, not UseIISIntegration.

Reply:
*Aye!* I see it. **Add UseIIS to CreateDefaultBuilder** https://github.com/aspnet/MetaPackages/pull/278/